dashboard/templates/dashboard.html

```diff
@@ -52,7 +52,7 @@ <h4>{{ learningResource.short_title }}</h4>
                 <h4>{{ learningResource.title }}</h4>
               {% endif %}
               <div style="padding: 2px;">{{ learningResource.department }}</div>
-              <a href="{{ resource.detail }}" 
+              <a onclick="window.location='{% url 'detail' learningResource.id %}';"
```

Comment:
Why do you need the window.location here? I just tried `<a onclick="{% url 'detail' learningResource.id %}"`.  It's more Django-like and works the same way! \o/

Reply:
I think it's a browser issue. I think to be on the save side it should be `onclick="javascript:window.location.href='{% url 'detail' learningResource.id %}';"`. Check this out: https://stackoverflow.com/questions/10252809/onclick-window-location-href-wont-work-in-chrome-or-safari

Reply:
Fixed it with commit #784226b